docs/extensibility.md

```diff
@@ -1,23 +1,22 @@
 # Extensibility
-While Akri has several [currently supported discovery protocols](./roadmap.md#currently-supported-protocols) and sample brokers and applications to go with them, the protocol you want to use to discover resources may not be implemented yet. This walks you through all the development steps needed to implement a new protocol and sample broker. It will also cover the steps to get your protocol and broker[s] added to Akri, should you wish to contribute them back. 
 
-To add a new protocol implementation, three things are needed:
-1. Add a new DiscoveryHandler implementation in the Akri Agent
-1. Update the Configuration CRD to include the new DiscoveryHandler implementation
-1. Create a protocol broker for the new capability
+While Akri has several [currently supported discovery protocols](./roadmap.md#currently-supported-protocols) and sample brokers and applications to go with them, the protocol you want to use to discover resources may not be implemented yet. This walks you through all the development steps needed to implement a new protocol and sample broker. It will also cover the steps to get your protocol and broker[s] added to Akri, should you wish to contribute them back.
 
-## The mythical Loch Ness resource
-To demonstrate how new protocols can be added, we will create a protocol to discover Nessie, a mythical Loch Ness monster that lives at a specific url.
+To add a new protocol implementation, several things are needed:
 
-For reference, we have created a [nessie branch](https://github.com/deislabs/akri/tree/nessie) with the implementation defined below.  For convenience, you can [compare the nessie branch with main here](https://github.com/deislabs/akri/compare/nessie).
+1. Add a new DiscoveryHandler implementation in Akri Agent
+1. Update Configuration CRD to include the new DiscoveryHandler implementation
+1. Build versions of Akri agent and controller that understand the new DiscoveryHandler
+1. Create a (protocol) Broker for the new capability

```

Comment:
I would say "(protocol) Broker Pod" since it hasnt been clarified that the broker is a Pod yet

Reply:
👍